docs/advanced/dcrtime.md

```diff
@@ -0,0 +1,121 @@
+cd90cc268d9ceef6e276bfa7a615c5f85b5a27b0d917ee3f1f1e5d5598f2fa00 Not anchored
+```
+
+If we wait until the transaction containing the merkle root has been mined into a block and call dcrtime again, the dcrtime server will verify that the transaction has been anchored, as well as provide the merkle path (path to the merkle root), the transaction ID (TxID) of the transaction containing the merkle root, and a `Chain Timestamp` providing the time the block was mined. 
+
+```
+$ dcrtime --testnet -v 8496855341883fdc90cc532f8304d1c46a60586fb15d99f07e41bb5ab19c79c6
+8496855341883fdc90cc532f8304d1c46a60586fb15d99f07e41bb5ab19c79c6 Verify
+8496855341883fdc90cc532f8304d1c46a60586fb15d99f07e41bb5ab19c79c6 OK
+  Chain Timestamp: 1497013614
+  Merkle Root    : 8496855341883fdc90cc532f8304d1c46a60586fb15d99f07e41bb5ab19c79c6
+  TxID           : 4172a560a7035c169c4da60cba2cb1fbac686bd01224e09a1a56ce5e6f31cff0
+```
+
+Note that the dcrtime client does verify the merkle path that the server returns. 
+
+Our file is now anchored in the Decred blockchain. Any third party can now prove that `myfile.txt` existed, in its exact form, at the time the block was mined (Chain Timestamp). 
+
+### Politeia
+
+Building on our simple example, the following example illustrates how dcrtime is used to timestamp data in Politeia.
+
+Politeia commits user data (new proposals, edits, etc.) as needed to a public git [repo](https://github.com/decred-proposals/mainnet). For performance reasons, incoming vote and comment data are stored in the Politeia backend in "journal" files. These journal files are then committed to the repo when other data such as a new proposal is submitted, or every hour if no other data is submitted. Once Politeia verifies that these commits are in the repo, dcrtime is used to anchor the git hash for each commit. For each hash anchored, an anchor file is committed to the `/anchors` directory in the repo. To fasciliate verification, metadata for each anchor is also appended to a single text file, `anchor_audit_trail.txt`, which provides a chonological record of anchors. 
+
+The diagram below provides a high-level diagram of the timestamping process.
+
+![dcrtime politeia](/img/dcrtime_politeia_diagram.png)
+
+To illustrate, let's examine an anchor from a real proposal, the [Decred Bug Bounty Proposal: Phase 2](https://proposals.decred.org/proposals/073694ed82d34b2bfff51e35220e8052ad4060899b23bc25791a9383375cae70) proposal. This anchor hashes two commits to the blockchain. The first [commit](https://github.com/decred-proposals/mainnet/commit/9125d351db4a429681cd7158d2c17d62a2b47c4c) contains all vote data submitted in the last hour. The second [commit](https://github.com/decred-proposals/mainnet/commit/afcca3b205ab6ec749d26e1903414aa35acd9767) contains all comments submitted in the last hour. First, Politeia will submit an Anchor commit, which appends an "Audit Trail Record" containing metadata to `anchor_audit_trail.txt`, as shown below.  
```

Comment:
The page uses a mix of "Anchor" and "anchor". Should probably standardize

Reply:
Went with lowercase anchor.